data/saas/config/mailchimp_transactional_config.yml

```diff
@@ -0,0 +1,46 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Mailchimp Transactional SaaS Config
+  type: mailchimp_transactional
+  description: A sample schema representing the Mailchimp Transactional (Mandrill) connector for Fides
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+      default_value: mandrillapp.com/api/1.0
+    - name: api_key
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: api_key
+      configuration:
+        body: |
+          {
+            "key": "<api_key>"
+          }
```

Comment:
Making `Mailchimp Transactional` a separate connector as it is a separate Mailchimp addon. Mailchimp purchased Mandrill, and the connector params and client config are different than the base mailchimp connector.